modules/button/css/lib/button.scss

```diff
@@ -7,104 +7,53 @@ $_wdc-btn-caret-down-icon-hover: svg-load(
   fill=$wdc-btn-icon-hover-color
 );
 
-@mixin wdc-btn-icon-inline($_icon) {
-  content: '';
-  background-image: $_icon;
-  background-position: center;
-  background-size: $wdc-spacing-s;
-  background-repeat: no-repeat;
-}
-
-@mixin wdc-btn-box-shadow($color) {
-  box-shadow: 0 0 0 $wdc-btn-border-size $color;
-}
+$_wdc-btn-caret-down-icon-white: svg-load(
+  '#{$_wdc-btn-icons-path}/wd-icon-caret-down.svg',
+  fill=$wdc-btn-icon-color-white
+);
 
 @mixin wdc-btn() {
   @include wdc-type-body();
   @include wdc-type-variant-btn();
   display: inline-block;
   text-decoration: none;
-  font-family: $wdc-font-family;
-  font-weight: $wdc-btn-font-weight;
-  min-width: $wdc-btn-min-width-l;
+  min-width: 96px;
   max-width: $wdc-btn-max-width-l;
   height: $wdc-btn-height-l;
-  padding: $wdc-btn-padding-l;
-  background-color: $wdc-btn-bg;
-  border: 1px solid $wdc-btn-border;
-  color: $wdc-btn-color;
-  border-radius: $wdc-btn-border-radius;
-  transition: all 120ms, box-shadow 100ms;
+  padding: 0 14px;
+  background-color: $wdc-btn-bg-secondary;
+  color: $wdc-btn-secondary-text-color;
+  border-radius: 999px;
+  transition: 0.2s background, 0.2s border-color, 0.2s box-shadow;
   text-align: center;
   font-size: 14px;
   cursor: pointer;
   box-sizing: border-box;
-  -webkit-font-smoothing: antialiased;
-  outline: none;
-  line-height: 38px;
-
-  .wd-icon {
-    height: $wdc-btn-icon-size-s;
-    width: $wdc-btn-icon-size-s;
-  }
+  vertical-align: middle;
+  border: 2px solid transparent;
+  font-weight: 700;
 
-  .wd-icon-background {
-    fill: transparent;
+  &.wdc-btn-hover,
+  &:hover {
+    background-color: $wdc-btn-bg-secondary-hover;
   }
 
-  .wd-icon-accent,
-  .wd-icon-fill {
-    fill: $wdc-btn-icon-color;
+  &.wdc-btn-active,
+  &:active {
+    background-color: $wdc-btn-bg-secondary-active;
```

Comment:
The focus styles are overriding this, so pressing the spacebar while focussed doesn't show any background color change. This is the case for all of the new buttons

Reply:
@anicholls i added this to accessibility.scss, do I still need these styles here? Can you double check, please, thanks!

Reply:
Looks good 👍